.github/workflows/compile-platform-examples.yml

```diff
@@ -53,27 +53,17 @@ jobs:
             stackMem: true
             dma: false
             alarmRtc: false
-          - fqbn: Infineon:arm:XMC1100_H_BRIDGE2GO
+          - fqbn: Infineon:xmc:XMC4200_Platform2GO

```

Comment:
Incorrect Board name gives TWO XMC4200 Platform 2 Go

Reply:
What do you mean? there is a XMC4**2**00 and XMC4**4**00 Platform 2Go.